tests/baselines/reference/valuesMergingAcrossModules.errors.txt

```diff
@@ -0,0 +1,28 @@
+tests/cases/conformance/externalModules/c.ts(1,10): error TS2440: Import declaration conflicts with local declaration of 'A'.
+tests/cases/conformance/externalModules/c.ts(6,1): error TS2349: This expression is not callable.
+  Type 'typeof A' has no call signatures.
+
+
+==== tests/cases/conformance/externalModules/a.ts (0 errors) ====
+    function A() {}
+    export { A };
+    
+==== tests/cases/conformance/externalModules/b.ts (0 errors) ====
+    import { A } from "./a";
+    type A = 0;
+    export { A };
+    
+==== tests/cases/conformance/externalModules/c.ts (2 errors) ====
+    import { A } from "./b";
+             ~
+!!! error TS2440: Import declaration conflicts with local declaration of 'A'.
+    namespace A {
+      export const displayName = "A";
+    }
+    
+    A();

```

Comment:
That’s right, but only because `A` is an alias. Normally, slightly disconcerting though it is, we allow a function to merge with an instantiated namespace: [playground](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAQQBQEpEG8CwAoRQxAJwFMoRiwBufAX3zAEMBbUgZwAcmJSVt8RRBATsoiACYwuAGyYBPAHKs+AXkQAiZBtp46QA). I only learned this in the course of my PR, but the rules are stricter for merges across module boundaries (thankfully). Two values are just not allowed to merge across modules.